app/common/directives/file-upload.directive.js

```diff
@@ -4,18 +4,27 @@ function FileUpload() {
     return {
         restrict: 'E',
         templateUrl: 'templates/common/directives/file-upload.html',
+        replace: true,
```

Comment:
I think it's ok, I checked and apparently it's not being removed prior to 2.x

I would argue to keep this because it has the nice effect of allowing the id to fall directly through to the input. Which is neater than having to pass the id as a param but I won't fight particularly hard for it.